x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -221,6 +220,15 @@ IDENTIFIER
     : (LETTER | '_' | '@') (LETTER | DIGIT | '_')*
     ;
 
+QUOTED_IDENTIFIER
+    : '`' ( ~'`' | '``' )* '`'
+    ;
+
+eventValue
+    : STRING
```

Comment:
does it make sense to use quoted identifier instead or in addition to this?

Reply:
I don't think so, it'a a string value so it should have the semantics of a string literal in case we encounter any non-letter/digit/_ chars there.